src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,24 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        contextMenuWasEnabled = atom.config.get('core.showAtomInShellContextMenu')
```

Comment:
> If this cannot be done, or cannot be done easily

Loading and accessing `atom.config`  from the browser process cannot be done easily at this time. `src/config.coffee` would need to be adjusted to work in the browser process where packages (and other things) are unavailable.